EC-84: Revise Configuration, Customers, eContent, Login and Logging (#5)
ezsystems/commerce-documentation: docs/guide/configuration/session_handling.md

# Session handling

The default session handling should be changed, because after clearing the cache all users will be logged out.  

> Session now are handled with PDO. You can also switch to Memcache. 

Reviewer: ```suggestion
Sessions are handled with PDO, but you can switch to Memcache. 
```

);
```

## Set up name for SiteAccesses

It is very important to set a session name. Otherwise eZ Platform will generate a unique name per SiteAccess

> which causes issues if you switch e.g. the language/SiteAccess and users cannot share a basket and login across SiteAccesses.

Reviewer: Is it "e.g. if you switch the language/SiteAccess, users will not be able to ..." or "e.g. if you switch, there will be issues with language/SiteAccess, and users will not be able to..."?
